static/src/javascripts/projects/common/modules/identity/upsell/consent-card/ConsentCard.js

```diff
@@ -1,14 +1,7 @@
 // @flow
 import React, { Component } from 'preact-compat';
 import { FollowButtonWrap } from 'common/modules/identity/follow/FollowButtonWrap';
-
-export type Consent = {
-    id: string,
-    name: string,
-    description: string,
-    isOptOut: boolean,
-    isChannel: boolean,
-};
+import type { Consent } from '../store/consents';
```

Comment:
is there a convention for relative vs absolute imports?

Reply:
absolute is better but IntelliJ does relative by default and sometimes it looks nicer, happy to standarise